### ISSUE TITLE
Song sorting not available

### ISSUE THREAD

**Issue by veraverto** [UltraStar-Deluxe/USDX#513]
**Please, do not create duplicate issues**

### Actual behaviour

When in song selection, pressing M, choosing "Sort" I get an error message "Not Available!".
<img width="1440" alt="Bildschirmfoto 2020-01-26 um 17 54 23" src="https://user-images.githubusercontent.com/18749953/73138609-f20bfa80-4064-11ea-9917-e8f3873a1ce1.png">

### Expected behaviour

Sort songs by artist, title or other values.

### Steps to reproduce

Start self compiled UltraStar Deluxe
Go to song selection
Press M
Choose "Sort"

### Details

Provide some additional information:

- USDX version: UltraStar Deluxe 2017.8.0
- Operating System + version: macOS 10.13.6

**Comment by tobi512** [comment]
Same error with the latest 2020.4.0 build, is there anything we can do to get this feature running?

**Comment by basisbit** [comment]
this menu entry is not implemented. Usually what you want to do is to configure song sorting (for example by artist/...) in the game settings, and then restart the game to apply these changes.

**Comment by tobi512** [comment]
Thanks for the tip, I didn't deep dive into all the settings yet and artist instead of edition seems to be the better default value indeed!
However, quick adjustment of the sort order via the menu would be great and probably not too much work to implement (from my general developers perspective, correct me if I'm wrong 😄).

**Comment by basisbit** [comment]
of course, feel free to create a pull request and change this objective pascal code ;-)

**Comment by dgruss** [comment]
was addressed in #869
